sharktank/sharktank/tools/eval_llm_vmfb.py

```diff
@@ -132,9 +150,27 @@ def main(
     parser.add_argument(
         "--min-context", help="required context length", type=int, default=0
     )
-    parser.add_argument("--iree-hal-target-device", help="Target device if compiling")
+    # iree-hal-target-device needs to support multiple devices
+    parser.add_argument(
+        "--iree-hal-target-device",
+        help="Target device(s) if compiling",
+        action="append",
+    )
     parser.add_argument("--iree-hip-target", help="Iree hip target")
+    parser.add_argument(
+        "--pipeline-parallelism-size",
+        help="Pipeline parallelism size",
+        type=int,
+        default=1,
+    )
     args = parser.parse_args()
+
+    # TODO: This is deceiving, it's the tokenizer directory
+    if args.tokenizer.endswith("tokenizer.json"):
```

Comment:
Better to throw errors for the right path rather than assuming json name. This just leads to weird tokenizer naming behavior.